bridge/README.md

```diff
@@ -27,6 +27,75 @@ publish the payload on to a specified "output."
 
 Both **senders** and **receivers** are defined in terms of their input, and optional JavaScript transformation, and their output.
 
+---
+
+Transformations are configured as either a single string of JS source code:
+
+```yaml
+transformation: |
+    function handler(input) {
+      return {
+        app_id: input.key,
+        message: {
+          eventType: input.event_type,
+          payload: input.data
+        }
+      };
+    }
+```
+In this situation, the input value will be pre-parsed as JSON.
+
+You can also configure transformations using the verbose form, allowing control of the `format`:
+```yaml
+transformation:
+  format: "json"
+  src: |
+    function handler(input) {
+      return {
+        app_id: input.key,
+        message: {
+          eventType: input.event_type,
+          payload: input.data
+        }
+      };
+    }
+```
+The `format` field can be set to either `string` (ie. the _raw payload_) or `json` (the default behavior).
```

Comment:
Not `object`? (Instead of `json`)? I feel like `json` implies it could be a `[...]`, or `1` too

Reply:
Yeah, this is tricky to explain (which is probably bad) but at the outer edge of the boundary, we only require that the thing parses as JSON. The transformation can be required to conform to a Map type.

This is to say, you can send valid json (of any shape) and so long as the transformation converts to `Object`, we're good.

In `#5762`, we'll be explicitly requiring a certain shape for the JSON regardless of whether or not it comes out of a queue or a transformation, but still the input and output aren't symmetrical.

Does that make any sense? 😰